ignite/config/chain/parse.go

```diff
@@ -102,6 +103,26 @@ func ReadConfigVersion(configFile io.Reader) (version.Version, error) {
 	return c.Version, err
 }
 
+// ReadProtoPath reads the build proto path from the config.
+func ReadProtoPath(configFile io.Reader) (string, error) {
+	c := struct {
```

Comment:
this feels a bit hacky tbh. don't we have an existing read util for reading the ignite config?

Reply:
Yes we should have

Reply:
we have it. But IMHO, we don't need to load the entire config file into the memory, for we only read one variable. And it's using the same logic from the `ReadConfigVersion` function

Reply:
The config file isn't that big, right? Is that really an issue?
We should at least then add a regression test to harden this behavior if we don't use the util function.

Reply:
The config file can also hold the genesis information, bigger than the usual config parameters. For instance, check the [SPN config file](https://github.com/tendermint/spn/blob/main/config.yml)

Reply:
wdyt? Do you agree to use this one or vote for the default parser?

Reply:
I am fine with this one if we add a test that verifies that they both return the same thing. This way if we break the config in the future we won't forget to update here too.

Reply:
It would be great if we figure out a way to the config lives globally for all these methods and we don't need to load in the memory more than one time